yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -352,15 +352,49 @@ func createMigrationAssessmentStartedEvent() *cp.MigrationAssessmentStartedEvent
 func createMigrationAssessmentCompletedEvent() *cp.MigrationAssessmentCompletedEvent {
 	ev := &cp.MigrationAssessmentCompletedEvent{}
 	initBaseSourceEvent(&ev.BaseEvent, "ASSESS MIGRATION")
-	report, err := json.Marshal(assessmentReport)
+
+	finalReport := AssessMigrationPayload{
+		AssessmentJsonReport: assessmentReport,
+		SourceSizeDetails: SourceDBSizeDetails{
+			TotalDBSize: source.DBSize,
+		},
+		TargetRecommendations: TargetSizingRecommendations{},
+		MigrationComplexity:   getMigrationComplexity(), //TODO: to figure out proper algorithm
+		ConversionIssues:      schemaAnalysisReport.Issues,
+	}
+	colocatedTables, err := assessmentReport.GetColocatedTablesRecommendation()
 	if err != nil {
-		utils.PrintAndLog("Failed to serialise the assessment report to json (ERR IGNORED): %s", err)
+		utils.PrintAndLog("Failed to get the colocated tables recommendation from assessmentReport: %v", err)
+	}
+	if assessmentReport.TableIndexStats != nil {
+		for _, stat := range *assessmentReport.TableIndexStats {
+			if stat.IsIndex {
+				finalReport.SourceSizeDetails.TotalIndexSize += *stat.SizeInBytes
+			} else {
+				finalReport.SourceSizeDetails.TotalTableSize += *stat.SizeInBytes
+				finalReport.SourceSizeDetails.TotalTableRowCount += *stat.RowCount
+				if slices.Contains(colocatedTables, fmt.Sprintf("%s.%s", stat.SchemaName, stat.ObjectName)) {
```

Comment:
hope we won't run into case sensitivity issues here? Maybe we should have represented it as NameTuples?